### ISSUE TITLE
Remove stylistic alternates and aalt feature code

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#211]
Project creative directors asked that we remove support for alternate glyph forms so that internal design teams do not have the opportunity to select from the alternate forms to define branding in their own product areas.

This change will occur as of the v4 release of Google Sans.  The source definitions will remain but the glyphs and feature code support will not be compiled into the production fonts.  As part of this transition we will eliminate the "default" and "expert" build targets and settle on a single default build that is compiled into the top level of the build/GoogleSans directory.

We are aware that some forms are **not** stylistic alternates (e.g., Denis Jacquerye's dagesh-hb.alt) and will need to remain in the build.  Full review of the required *.alt glyphs to follow.

**Comment by chrissimpkins** [comment]
@danielgrumer The `aalt` feature is subset out in the Fonts API so this is not available to anyone using the web fonts.  We are going to remove all other `aalt` feature support in the v4 release.  Can the justification alternates be removed from the `aalt` feature block so that we can get rid of `aalt` altogether?

**Comment by chrissimpkins** [comment]
I reviewed all of the *.alt alternate forms.  The following alternate style glyphs can be set to not export / not compile into fonts:

```python
nonexport_list = ['Ccedilla.alt', 'G.alt', 'Gbreve.alt', 'Gcircumflex.alt', 'Gcommaaccent.alt', 'Gdotaccent.alt', 'I.alt', 'Iacute.alt', 'Ibreve.alt', 'Icircumflex.alt', 'Idieresis.alt', 'Idotaccent.alt', 'Igrave.alt', 'Imacron.alt', 'Iogonek.alt', 'Itilde.alt', 'J.alt', 'Jcircumflex.alt', 'K.alt', 'Kcommaaccent.alt', 'M.alt', 'Q.alt', 'R.alt', 'Racute.alt', 'Rcaron.alt', 'Rcommaaccent.alt', 'Scedilla.alt', 'W.alt', 'Wacute.alt', 'Wcircumflex.alt', 'Wdieresis.alt', 'Wgrave.alt', 'a.alt', 'aacute.alt', 'abreve.alt', 'abreveacute.alt', 'abrevedotbelow.alt', 'abrevegrave.alt', 'abrevehookabove.alt', 'abrevetilde.alt', 'acircumflex.alt', 'acircumflexacute.alt', 'acircumflexdotbelow.alt', 'acircumflexgrave.alt', 'acircumflexhookabove.alt', 'acircumflextilde.alt', 'adieresis.alt', 'adotbelow.alt', 'ae.alt', 'aeacute.alt', 'agrave.alt', 'ahookabove.alt', 'amacron.alt', 'ampersand.alt', 'aogonek.alt', 'aring.alt', 'aringacute.alt', 'atilde.alt', 'caron.alt', 'caroncomb.alt', 'caroncomb.alt.cap', 'ccedilla.alt', 'cedilla.alt', 'cedillacomb.alt', 'comma.alt', 'copyright.alt', 'f_f_j.alt', 'f_f_k.alt', 'f_f_t.alt', 'f_j.alt', 'f_k.alt', 'f_t.alt', 'j.alt', 'jcircumflex.alt', 'jdotless.alt', 'k.alt', 'kcommaaccent.alt', 'nine.alt', 'nine.alt.cap', 'one.alt', 'one.alt.cap', 'published.alt', 'q.alt', 'quotedblbase.alt', 'quotedblleft.alt', 'quotedblright.alt', 'quoteleft.alt', 'quoteright.alt', 'quotesinglbase.alt', 'r.alt', 'r_t.alt', 'racute.alt', 'rcaron.alt', 'rcommaaccent.alt', 'registered.alt', 'scedilla.alt', 'semicolon.alt', 'servicemark.alt', 'servicemark.alt2', 'servicemark.alt3', 'seven.alt', 'seven.alt.cap', 'six.alt', 'six.alt.cap', 't.alt', 't_f.alt', 't_t.alt', 'tbar.alt', 'tcaron.alt', 'tcedilla.alt', 'tcedilla.alt.2', 'tcommaaccent.alt', 'trademark.alt', 'trademark.alt2', 'trademark.alt3', 'y.alt', 'yacute.alt', 'ycircumflex.alt', 'ydieresis.alt', 'ydotbelow.alt', 'ygrave.alt', 'yhookabove.alt', 'ytilde.alt', 'zero.alt', 'zero.alt.cap']
```

**Comment by chrissimpkins** [comment]
We need to keep the `dagesh-hb.alt` (`uni05BC.alt`) glyph that was added in #134. This is currently in the master branch but **not** in David Berlow's source files.

**Comment by danielgrumer** [comment]
> Can the justification alternates be removed from the `aalt` feature block so that we can get rid of `aalt` altogether?

yes, it's ok to remove it.

**Comment by danielgrumer** [comment]
> We need to keep the `dagesh-hb.alt` (`uni05BC.alt`) glyph that was added in #134. This is currently in the master branch but **not** in David Berlow's source files.

This alternative dagesh-hb.alt appears very few times, and is only a smaller dot. 
I believe this component can be flatten to outlines if you need to remove it too.

**Comment by moyogo** [comment]
>> We need to keep the dagesh-hb.alt (uni05BC.alt) glyph that was added in #134. This is currently in the master branch but not in David Berlow's source files.

> This alternative dagesh-hb.alt appears very few times, and is only a smaller dot.
I believe this component can be flatten to outlines if you need to remove it too.

Maybe `dagesh-hb.alt` can be renamed instead. It was turned into its own glyph used as a component to save on file size as it had the same contour in all the glyphs it was used.

**Comment by chrissimpkins** [comment]
> > Can the justification alternates be removed from the `aalt` feature block so that we can get rid of `aalt` altogether?
> 
> 
> 
> yes, it's ok to remove it.

Thanks Daniel!

**Comment by chrissimpkins** [comment]
> Maybe `dagesh-hb.alt` can be renamed instead. It was turned into its own glyph used as a component to save on file size as it had the same contour in all the glyphs it was used.

Not a huge issue IMO. We just want to make sure that required glyphs don't land in the non-export bin with the other .alt glyphs.  I believe that this is the only one.

**Comment by madig** [comment]
Non-export glyphs are not exported, but are decomposed into glyphs where they are actually used. To save bytes, only stuff you truly don't want should be non-exported.

**Comment by chrissimpkins** [comment]
> To save bytes, only stuff you truly don't want should be non-exported.

Apologies.  This was what I was trying to convey.  We want to exclude all *.alt glyphs in the https://github.com/googlefonts/googlesans/issues/211#issuecomment-785516852 list from compiling into the binary in any composed or decomposed form.  I *think* that we truly don't want the ones in the list to exist at all in the fonts.  Please let me know if you come across any areas where they are used outside of `aalt` stylistic alternate support.

We **do want** dagesh-hb.alt.  It is used as a component in a number of the Hebrew forms as of #134.

**Comment by madig** [comment]
Then set *.alt except dagesh-hb.alt to non-export 👍. Once we get to merging stuff, I can see if *.alt glyphs are used anywhere else, like components.

**Comment by chrissimpkins** [comment]
Removed in https://github.com/googlefonts/googlesans/pull/240/commits/505d69c1d708b2c831eb5d9a1e9ccab6afe266c2

This will be merged in #240
